inc/admin/classes/class-wp-statuses-admin.php

```diff
@@ -201,8 +201,11 @@ public function register_script( $press_this = '' ) {
 	public function add_meta_box( $post_type, $post ) {
 		global $publish_callback_args;
 
-		// Bail if the Post Type is not supported.
-		if ( ! wp_statuses_is_post_type_supported( $post_type ) ) {
+		// Bail if the Post Type is not supported, or if post uses Gutenberg block editor
+		if (
+			! wp_statuses_is_post_type_supported( $post_type )
+			|| use_block_editor_for_post( $post )
```

Comment:
As the plugin main file has this [header tag](https://github.com/imath/wp-statuses/blob/master/wp-statuses.php#L7): `Requires at least: 4.7` and `use_block_editor_for_post()` was introduced in 5.0.0, could you add a `function_exists()` check before using it ?

eg : `|| ( function_exists( 'use_block_editor_for_post' ) && use_block_editor_for_post( $post ) )`

Reply:
Good point - pull request updated.